utils/vlog/vlog_to_model.py

```diff
@@ -174,7 +176,7 @@ def is_registered_path(tmod, pin, pout):
             outfile=outfile,
             xptr="xpointer(models/child::node())"
         )
-if True:
+else:
```

Comment:
This should not be `else` statement. This change is causing the `multiple_instance` test failures.

Reply:
this is actually correct